similarity/README.md

```diff
@@ -2,7 +2,59 @@
 
 This module specifies a simple API for NLP components that compare
 objects -- especially Strings -- and return a score indicating how
-similar they are.  It is used in our WordNet-, Named Entity-, embedding-, 
-and paraphrase-based similarity code to simplify integration of 
+similar they are.  It is used in our WordNet-, Named Entity-, embedding-,
+and paraphrase-based similarity code to simplify integration of
 different similarity resources.
- 
\ No newline at end of file
+
+## Download Resource File
+
+To download resource file including: Word2vec, Glove, ESA, simply run the main function in Datastore class in `edu.illinois.cs.cogcomp.sim` package. It will download those file from Cogcomp server to `src/main/resources/1.5` in your local machine. You can also comment out the resource that you don't want or use your own file. Notice: "paragram" is already included in the `src/main/resources/`.
+
```

Comment:
> the file will download into System.getProperty("user.home"), but I can't specify "user.home" locations in my configuration file. 

And by what I said before, you don't have to know about "user.home", because datastore gives your the file.